src/anomalib/pipelines/tiled_ensemble/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""Tiled ensemble pipelines."""
```

Comment:
Would it be an idea to add a detailed docstring here to give proper credit to you and your work? Something like the following 
https://github.com/open-edge-platform/anomalib/blob/5f37f74fb48c609f9bd8f0fdfaa0ada8e3834291/src/anomalib/data/datamodules/image/mvtecad2.py#L1-L28